spec/system/features_spec.rb

```diff
@@ -1,37 +0,0 @@
-# frozen_string_literal: true
```

Comment:
I removed this test because we now will use the `admin` role to determine whether to load the features page or not, instead of determining whether a user is an LTDS user based on env variable `LTDS_ADMIN_NET_ID_LIST`. Please refer to feature spec file `spec/features/flipflop_dashboard_spec.rb`